launcher/verifier/rest/rest.go

```diff
@@ -157,6 +157,14 @@ func convertRequestToREST(request verifier.VerifyAttestationRequest) *v1alpha1.V
 	}
 }
 
+func handleError(err error, message string) error {
+	const invalidArgument = "invalid argument"
+	if strings.Contains(err.Error(), invalidArgument) {
+		return fmt.Errorf(message+": %w", err)
+	}
+	return fmt.Errorf(message)
```

Comment:
I'm not sure we should completely drop non-InvalidArgument errors. An UNAVAILABLE error gives some information (we're down) although it is not actionable. Even an internal error will return an ID that we can use to help debug. This is probably where we rely mostly on the server to filter out implementation details and we can augment here based on error code